README.md

```diff
@@ -288,7 +288,14 @@ When running in Openshift, we need to grant the appropriate security context for
    oc adm policy add-scc-to-user privileged system:serviceaccount:submariner:submariner-routeagent
    oc adm policy add-scc-to-user privileged system:serviceaccount:submariner:submariner-engine 
    ```
-   
+
+### Flannel Notes
+
+When using flannel CNI, configure reverse path filtering to loose mode on the non-gateway nodes (i.e., worker nodes).
```

Comment:
I think those notes aren't necessary anymore right? you do it in code. I'm going to update this.